docs/en/operations/system-tables.md

```diff
@@ -487,7 +487,16 @@ Columns:
 
 -   `name` (`String`) – Name of the data part.
 
--   `active` (`UInt8`) – Flag that indicates whether the data part is active. If a data part is active, it’s used in a table. Otherwise, it’s deleted. Inactive data parts remain after merging.
+-   `part_type` ([String](../sql-reference/data-types/string.md)) — The data part storing format.
+
+    Possible Values:
+
+    -   `Wide` — each column is stored in a separate file on disk. 
+    -   `Compact` — all columns are stored in one file on disk. 
```

Comment:
```suggestion
    -   `Wide` — Each column is stored in a separate file on a disk. 
    -   `Compact` — All columns are stored in one file on a disk. 
```

Reply:
ClickHouse can store data in disks, in external and distributed storages, so "filesystem" is a more appropriate term to describe where data are stored.